app/statistics/template.hbs

```diff
@@ -107,7 +107,10 @@
             on-change = (action (mut toDate))
           }}
         {{/fs.label}}
-        {{date-buttons presetDate=(action (mut fromDate))}}
+        {{date-buttons
+            presetFromDate=(action (mut fromDate))
+            presetToDate=(action (mut toDate))
```

Comment:
If I understand correctly, `presetToDate` is an action. However the naming suggests otherwise. Please rename this so the type of the argument is clear, e.g `onUpdateToDate`